components/ProblemEditor/CodeEditorForm/CodeEditorForm.tsx

```diff
@@ -202,6 +210,27 @@ export const CodeEditorForm = ({ formRef }: Props) => {
                 />
               </Paper>
             </Box>
+            <Box flexGrow={1} display="flex" flexDirection="column">
```

Comment:
This creates the UI, and I assume there will be a subsequent PR on the backend to store the solution?

Reply:
yes, this was just the frontend part, Benny and I are currently working on the backend portion of this feature